userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -363,39 +363,37 @@ bool docker::resolve(sinsp_container_manager* manager, sinsp_threadinfo* tinfo,
 
 	tinfo->m_container_id = container_id;
 
-	existing_container_info = manager->get_container(container_id);
+	container_info = manager->get_container(container_id);
```

Comment:
well darn; does this break the whole model I wanted?

I was hoping that there wasn't a reason to get a mutable container_info outside of the manager

Reply:
We don't modify the container we got from the manager here. I'll try to structure the delayed cgroup lookups in a way that doesn't do that either (it will hopefully hold onto a copy of the mutable shared_ptr from before adding it to the manager). That does introduce a data race but I believe it'll be harmless enough (we're only writing ints from another thread) -- we can either discuss this now or in that (near future) PR